packages/auto-drive/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './cid/index.js'
```

Comment:
I believe you could remove all the .js

Reply:
I tried, but since I'm using ES modules due to the dependencies of the package. `moduleResolution` is forced to `NodeNext` and it requires having extension.